### ISSUE TITLE
Accessing Devices on local network

### ISSUE THREAD

**Issue by MatGriffin** [pivpn/pivpn#187]
All installs nicely, can connect via OpenVPN client on 4G iPad device and see my IP is 10.0.8.2 and browse internet.

I can connect to other devices on my network like my other rapsberry pi running Apache at 192.168.1.10

I'm try to send a WOL command to a PC in a S5 state. I know this works when I'm on the same network but it will not work when via the VPN, any idea why

**Comment by redfast00** [comment]
Since WOL packets go over the data layer (layer 2). Our current default is a layer-3 based IP tunnel, so the WOL packets don't work. OpenVPN does support a layer-2 based Ethernet TAP, so it might work if you convert the OpenVPN server to TAP instead of TUN. Another option is to install [etherwake](https://linux.die.net/man/8/ether-wake) on the Pi and then SSH into it with you iPad and send the command with the Pi. Let me know if you get it working.

**Comment by MatGriffin** [comment]
I got this to work via wakeonlan and PHP.

Simple php script that calls the wakeonlan command on the Pi which sends the magic packet to the MAC address of the PC.

`<?php
   exec('wakeonlan D1:15:C3:E3:15:9A');
?>`

When awake I can remote in. I also added Airytec Switch off to the PC. Once I've finished my remote sessions I then go to the IP of the PC and choose one of the options, sleep hibernate etc.
